inst/extdata/energy/A44.flsp_bm2_state_res.csv

```diff
@@ -0,0 +1,56 @@
+# File: A44.flsp_bm2_state_res.csv
+# Title: Residential floorspace by state
+# Units: Billion m2
+# Source: Written by L111_Res_flsp.R; RGCAM output
```

Comment:
This is the csv file that is i the assumptions folder of the energy-data but written by a level 1 GCAM-USA file, but maybe not the current version of that code. In any case, I think the file name L111_Res_flsp.R may be obsolete, as is the name RGCAM.  So, I would change the comment here.  There is a file with this name written by GCAM-USA\Level1\LA144.Residential.R.   @bpbond again, have you seen this type of thing before?  Does @pkyle  know?  @gokuliyer ?